packages/flutter_tools/test/general.shard/dart/pub_get_test.dart

```diff
@@ -31,6 +31,10 @@ void main() {
     Cache.flutterRoot = getFlutterRoot();
   });
 
+  tearDown(() {
+    MockDirectory.findCache = false;
```

Comment:
This gets set to true in some tests and never set back to false.  False is default.

Reply:
(this mock is local to this file)